app/src/main/java/com/example/parkingsystem/mvp/contract/ParkingContract.java

```diff
@@ -1,16 +1,24 @@
 package com.example.parkingsystem.mvp.contract;
 
+import com.example.parkingsystem.fragment.ListenerDialogFragment;
+
 public interface ParkingContract {
 
     interface Model {
-        void setNumber();
+        void setSpaces(int spaces);
+
```

Comment:
remove the extra spaces between method declaration pls

Reply:
done